src/app/routes/topic/getInitialData/index.js

```diff
@@ -1,8 +1,14 @@
-export default () => {
+import { fablPath } from '../../../../../.secret';
```

Comment:
This will be removed before merge and fablPath will be accessed via an environment variable

Reply:
How are you planning engineers provide the path via an environment variable without committing it?

Reply:
I've stopped using the secret folder locally and pass in the environment variable on startup